.circleci/config.yml

```diff
@@ -0,0 +1,38 @@
+version: 2
+
+jobs:
+  build:
+    docker:
+      - image: cimg/python:3.7
+    steps:
+      - checkout
+
+      - restore_cache:
+          keys:
+            - deps-{{ checksum "poetry.lock" }}
+
+      - run:
+          name: Install Dependencies
+          command: poetry install
+
+      - save_cache:
+          key: deps-{{ checksum "poetry.lock" }}
+          paths:
+            - /home/circleci/.cache/pypoetry/virtualenvs
+
+      - run:
+          name: Run pylint
+          command: poetry run pylint symphony_api_client_python tests
+
+      - run:
+          name: Run tests with coverage
+          command: poetry run pytest
+
+      - store_test_results:
+          path: junit.xml
+
+      - store_artifacts:
```

Comment:
isn't store_test_results enough to view the test results?

Reply:
I followed this: https://circleci.com/docs/2.0/collect-test-data/#pytest
And test is not reported under the test tabs.

Reply:
It looks like path is a folder and not directly the junit.xml file

Reply:
I think it should be a folder, it says `path: test-results` but not a file.

Reply:
`Note: Please write your tests to subdirectories of your store_test_results path, ideally named to match the names of your particular test suites` - [source](https://circleci.com/docs/2.0/configuration-reference/#store_test_results)